index.js

```diff
@@ -70,21 +71,31 @@ function lint(textEditor) {
 			};
 		}
 
+		// taken from linter-eslint
+		let range;
+		const msgLine = x.line - 1;
+		try {
```

Comment:
The try/catch should be around the `rangeFromLineNumber` call. No point in try/catching more than necessary.